### PR TITLE
created create guild button which opens a dialog that allows you to create a created guild with the create guild api in create created create guild

### PR DESCRIPTION
**Pull request by aspectxlol**

### CODE REVIEW

**Comment by aspectxlol** [comment]
src/components/dialogs.tsx
src/pages/index.tsx

File: package.json
Comment (aspectxalt) [inline package.json:1]: should remove package manager, since the project doesnt use yarn

File: src/components/dialogs.tsx
Comment (aspectxalt) [inline src/components/dialogs.tsx:1]: change the default color from white to a darker color to attract more basement adults

File: start-database.sh
Comment (aspectxalt) [inline start-database.sh:1]: should remove this file, since the project uses Pterodactyl

**Review by aspectxalt — CHANGES_REQUESTED** [review]

File: src/components/dialogs.tsx
Comment (aspectxalt) [reply src/components/dialogs.tsx:1]: project owner did not specify a color theme, default was white

Comment (aspectxlol) [reply src/components/dialogs.tsx:1]: project owner did not specify a color theme, white was the default color theme

File: start-database.sh
Comment (aspectxlol) [reply start-database.sh:1]: pull author does not have a pterodactyl setup, therefor pull author relies on docker to setup a working postgresql database

**Comment by aspectxlol** [comment]
@EmirhanTr3 pls review

**Comment by EmirhanTr3** [comment]
Added in [3c1b208](https://github.com/EmirhanTr3/Whitespace/commit/3c1b2080e9df83991029cb1309c9c38ee1f88bc4)

**Pull request closed without merging**